stable/redis-ha/values.yaml

```diff
@@ -21,10 +21,37 @@ serviceAccount:
   ## If not set and create is true, a name is generated using the redis-ha.fullname template
   # name:
 
+## Enables a HA Proxy for better LoadBalancing / Sentinel Master support. Automatically proxies to Redis master.
+## Recommend for externally exposed Redis clusters.
+## ref: https://cbonte.github.io/haproxy-dconv/1.9/intro.html
+haproxy:
+  enabled: true
```

Comment:
I'm totally okay with this enabled by default, however you gotta keep in mind backwards compatibility.  So for me upgrading my Redis to this new version would break all my services because they rely on accessing the previously established service name and sentinel ports. I think my previous comment should address that.

Reply:
Sorry, this is supposed to be disabled by default. Thanks for noticing.